deepchecks/tabular/checks/train_test_validation/train_test_feature_drift.py

```diff
@@ -134,10 +135,12 @@ def __init__(
         self.max_num_categories_for_drift = max_num_categories_for_drift
         self.max_num_categories_for_display = max_num_categories_for_display
         self.show_categories_by = show_categories_by
-        if sort_feature_by in {'feature importance', 'drift score'}:
+        if sort_feature_by in {'feature importance', 'drift score', 'drift + importance'}:
```

Comment:
Not really important but who the hell did this as a set instead of a list? Such a weird decision